src/components/ErrorBoundary/index.tsx

```diff
@@ -174,7 +174,7 @@ const updateServiceWorkerInBackground = async () => {
 export default function ErrorBoundary({ children }: PropsWithChildren): JSX.Element {
   return (
     <Sentry.ErrorBoundary
-      fallback={Fallback}
+      fallback={({ error, eventId }) => <Fallback error={error} eventId={eventId} />}
```

Comment:
this is required to allow using hooks in a Fallback component (ErrorBoundary is a class component)